apis/v1alpha1/parameter_group.go

```diff
@@ -0,0 +1,89 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// Code generated by ack-generate. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	ackv1alpha1 "github.com/aws-controllers-k8s/runtime/apis/core/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ParameterGroupSpec defines the desired state of ParameterGroup.
+//
+// Represents the output of a CreateParameterGroup operation. A parameter group
+// represents a combination of specific values for the parameters that are passed
+// to the engine software during startup.
+type ParameterGroupSpec struct {
+	// An optional description of the parameter group.
+	Description *string `json:"description,omitempty"`
+	// The name of the parameter group family that the parameter group can be used
+	// with.
+	// +kubebuilder:validation:Required
+	Family *string `json:"family"`
+	// The name of the parameter group.
+	// +kubebuilder:validation:Required
+	ParameterGroupName *string `json:"parameterGroupName"`
```

Comment:
Please rename this to just "name". See [the s3 controller](https://github.com/aws-controllers-k8s/s3-controller/blob/main/generator.yaml#L103-L113) for an example